CMakeLists.txt

```diff
@@ -140,7 +144,34 @@ if(AVIF_LOCAL_LIBYUV)
         find_package(JPEG REQUIRED)
         set(LIBYUV_LIBRARY ${LIBYUV_LIBRARY} ${JPEG_LIBRARY})
     endif()
+    set(libyuv_FOUND TRUE)
```

Comment:
Note the libyuv version check at line 157. The new code here assumes the locally-built libyuv in ext/libyuv has a new enough version. I think it is fine to assume this. I just wanted to point out this difference from the original code.

Reply:
Right, I assumed the user would use the recommended commands.